apps/condo/domains/banking/schema/BankSyncTask.js

```diff
@@ -135,10 +145,60 @@ const BankSyncTask = new GQLListSchema('BankSyncTask', {
         },
     },
     hooks: {
+        resolveInput: async ({ resolvedData, context, operation }) => {
+            if (operation === 'create' && get(resolvedData, 'options.type') === 'SBBOL'){
+                const dateFrom = dayjs(get(resolvedData, 'options.dateFrom')).format('YYYY-MM-DD')
+                const dateTo = dayjs(get(resolvedData, 'options.dateTo')).format('YYYY-MM-DD')
+                if (!dateFrom || !dateTo || dateFrom === 'Invalid Date' || dateTo === 'Invalid Date') {
+                    throw new GQLError(INVALID_DATE, context)
+                }
+                resolvedData.options.dateFrom = dateFrom
+                resolvedData.options.dateTo = dateTo
```

Comment:
```suggestion
                const dateFrom = dayjs(get(resolvedData, 'options.dateFrom'))
                const dateTo = dayjs(get(resolvedData, 'options.dateTo'))
                if (!dateFrom.isValid() || !dateTo.isValid()) {
                    throw new GQLError(INVALID_DATE, context)
                }
                resolvedData.options.dateFrom = dateFrom.format('YYYY-MM-DD')
                resolvedData.options.dateTo = dateTo.format('YYYY-MM-DD')
```

Reply:
Why not to let API consumer to directly enter already formatted values in 'YYYY-MM-DD' format and move all the code to `validateInput`?